### PR TITLE
a3-press-release

### PR DESCRIPTION
**Pull request by Juanmediina**
Creacion de la pagina A3 press release

### CODE REVIEW

File: src/jm/html-css/a3-press-release/index.html [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+      src="https://www.apple.com/newsroom/images/product/apple-tv-plus/standard/Apple-Emmy-winner-Sept-2022-Ted-Lasso_big.jpg.large_2x.jpg"
+    />
+    <div class="text-grey">
+      <p>
+        “Ted Lasso” took home four wins at the 74th Primetime Emmy Awards,
+        including back-to-back wins for Outstanding Comedy Series for both its
+        first and second seasons.
+      </p>
+    </div>
+    <div class="article">
```
Comment (Iregui44) [inline src/jm/html-css/a3-press-release/index.html:39]: I think you whould try ang use more html tags and less div tags, Lets say article tag may be great here.

File: src/jm/html-css/a3-press-release/style.css [hunk 2]
```diff
@@ -0,0 +1,62 @@
+body {
+  display: block;
+  margin: 130px;
+}
+.text-grey {
```
Comment (Iregui44) [inline src/jm/html-css/a3-press-release/style.css:5]: Space?

File: src/jm/html-css/a3-press-release/style.css [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  line-height: 1.1666666667;
+  font-weight: 500;
+  letter-spacing: 0.009em;
+  font-family: "SF Pro Display", "SF Pro Icons", "Helvetica Neue", "Helvetica",
+    "Arial", sans-serif;
+  display: block;
+  margin-left: 140px;
+  margin-right: 140px;
+}
+img {
```
Comment (Iregui44) [inline src/jm/html-css/a3-press-release/style.css:36]: Space?

File: src/jm/html-css/a3-press-release/style.css [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+  max-width: 100%;
+  height: auto;
+  display: inline-block;
```
Comment (Iregui44) [inline src/jm/html-css/a3-press-release/style.css:39]: Maybe add margin? I dont see the img in the center, or any other part for waht matters.

File: src/jm/html-css/a3-press-release/style.css [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+  line-height: 1.3333733333;
+  font-weight: 400;
+  letter-spacing: -0.01em;
+  font-family: "SF Pro Text", "SF Pro Icons", "Helvetica Neue", "Helvetica",
+    "Arial", sans-serif;
+  margin-left: 140px;
+  margin-right: 140px;
+}
+
+.article {
```
Comment (Iregui44) [inline src/jm/html-css/a3-press-release/style.css:54]: Review this styles as the page dont look anything as the example

File: src/jm/html-css/a3-press-release/index.html [hunk 6]
```diff
@@ -0,0 +1,253 @@
...
+        Mac, popular smart TVs from Samsung, LG, Sony, VIZIO, TCL and others,
+        Roku and Amazon Fire TV devices, Chromecast with Google TV, PlayStation
+        and Xbox gaming consoles, and at tv.apple.com, for $4.99 per month with
+        a seven-day free trial. For a limited time, customers who purchase and
+        activate a new iPhone, iPad, Apple TV, Mac, or iPod touch can enjoy
+        three months of Apple TV+ for free.
+      </p>
+    </div>
+  </body>
+</html>
```
Comment (Iregui44) [inline src/jm/html-css/a3-press-release/index.html:253]: Missing press contact

File: src/jm/html-css/a3-press-release/style.css [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+  font-weight: 700;
+  letter-spacing: -0.003em;
+  font-family: "SF Pro Display", "SF Pro Icons", "Helvetica Neue", "Helvetica",
+    "Arial", sans-serif;
+  display: block;
+  margin-left: 140px;
+  margin-right: 140px;
+}
+
+h2 {
```
Comment (Iregui44) [inline src/jm/html-css/a3-press-release/style.css:25]: Review this styles as the page dont look anything as the example

**Review by Iregui44 — CHANGES_REQUESTED** [review]

File: src/jm/html-css/a3-press-release/index.html [hunk 8]
```diff
@@ -0,0 +1,253 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>A3</title>
+    <link rel="stylesheet" href="style.css" />
```
Comment (Iregui44) [inline src/jm/html-css/a3-press-release/index.html:7]: File usually called styles. Just a TOC :)

File: src/jm/html-css/a3-press-release/index.html [hunk 9]
```diff
@@ -0,0 +1,253 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
[the remark this answers is not in the document]
Comment (Juanmediina) [reply src/jm/html-css/a3-press-release/index.html]: change made

File: src/jm/html-css/a3-press-release/index.html [hunk 10]
```diff
@@ -0,0 +1,253 @@
...
+
+    <h2>
+      “Ted Lasso” once again becomes the most Emmy Award-winning comedy <br />
+      Apple TV+ lands nine Emmys total, including four wins for “Ted Lasso,” and
+      honors for “Carpool Karaoke: The Series,” “Severance,” “Schmigadoon!,” and
+      “Home Before Dark”
+    </h2>
+
+    <img
+      src="https://www.apple.com/newsroom/images/product/apple-tv-plus/standard/Apple-Emmy-winner-Sept-2022-Ted-Lasso_big.jpg.large_2x.jpg"
```
[the remark this answers is not in the document]
Comment (Juanmediina) [reply src/jm/html-css/a3-press-release/index.html:32]: added attribute

File: src/jm/html-css/a3-press-release/index.html (hunk 1 above)
Comment (Juanmediina) [reply src/jm/html-css/a3-press-release/index.html]: the label change has been made

File: src/jm/html-css/a3-press-release/style.css (hunk 2 above)
Comment (Juanmediina) [reply src/jm/html-css/a3-press-release/style.css:5]: ok

File: src/jm/html-css/a3-press-release/style.css (hunk 3 above)
Comment (Juanmediina) [reply src/jm/html-css/a3-press-release/style.css:35]: ok

File: src/jm/html-css/a3-press-release/style.css (hunk 4 above)
Comment (Juanmediina) [reply src/jm/html-css/a3-press-release/style.css:38]: added margin

File: src/jm/html-css/a3-press-release/style.css (hunk 5 above)
Comment (Juanmediina) [reply src/jm/html-css/a3-press-release/style.css]: corrections made

File: src/jm/html-css/a3-press-release/index.html (hunk 6 above)
Comment (Juanmediina) [reply src/jm/html-css/a3-press-release/index.html:260]: the contact press has been created

File: src/jm/html-css/a3-press-release/style.css (hunk 7 above)
Comment (Juanmediina) [reply src/jm/html-css/a3-press-release/style.css:25]: corrections made

**Review by Iregui44 — APPROVED** [review]

**Pull request merged**